MPChartLib/src/com/github/mikephil/charting/charts/RadarChart.java

```diff
@@ -358,10 +358,7 @@ public void setDrawWeb(boolean enabled) {
      */
     public void setSkipWebLineCount(int count) {
 
-        if (count < 0)
-            count = 0;
-
-        mWebModulus = count + 1;
+        mSkipWebLineCount = Math.max(0, count);
```

Comment:
I changed it to add 1 before the loop

The user says "skip 1 line", and you take that value and add 1 in the
setter, and subtract 1 in the getter - which makes nosense. Why not jusr
store the value...

‏בתאריך יום חמישי, 1 באוקטובר 2015, Philipp Jahoda <notifications@github.com>
כתב:

> In MPChartLib/src/com/github/mikephil/charting/charts/RadarChart.java
> <https://github.com/PhilJay/MPAndroidChart/pull/1058#discussion_r40932201>
> :
>
> > @@ -358,10 +358,7 @@ public void setDrawWeb(boolean enabled) {
> >       */
> >      public void setSkipWebLineCount(int count) {
> >
> > -        if (count < 0)
> > -            count = 0;
> > -
> > -        mWebModulus = count + 1;
> > +        mSkipWebLineCount = Math.max(0, count);
>
> Can't that result in 0?
> if someone sets the skipWebLineCount to 0 - then the loop that skips
> weblines has += 0
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PhilJay/MPAndroidChart/pull/1058/files#r40932201>.
>